textual_fspicker/base_dialog.py

```diff
@@ -100,11 +112,14 @@ def compose(self) -> ComposeResult:
         """
         with Dialog() as dialog:
             dialog.border_title = self._title
+            currentDrive = str(os.path.splitdrive(MakePath.of(self._location).expanduser().absolute())[0]+'\\').upper()
+            yield Select.from_values(self.drives, value=currentDrive)
```

Comment:
This would seem to force a drive selection widget on every platform; I'm not sure this is really a good design decision. If I were attempting this I suspect I'd only have this widget in place when on a system where it's relevant.